### ISSUE TITLE
Compatibility Blender 3.0.0

### ISSUE THREAD

**Issue by Queerdog** [Defaultio/BlenderMoco#1]
Hi Josh,
thank you so much for this great Add-on and all the hard work and tinkering you put into it!
I'm new to Blender and jumped into it just because of your add-on ;)

Currently I'm trying to get the add-on to work properly in version 3.0.0
I think I have solved all problems, but since I am an absolute beginner here, I might be missing something.

1.  I increased the version in line 5 to `"blender": (3, 0, 0) `
-> This got rid of the version error in the add-on menu 

2. I changed line 410 to `bl_region_type = 'UI'`
because apparently tabs have moved to the right side of the viewport.

ICON definitions have changed (that's why the UI was not drawn)
I picked new icons from here:
https://docs.blender.org/api/current/bpy.types.UILayout.html

3.  Line 459 to `layout.label(text="Axis Definitions", icon = 'WORKSPACE')`
4. Line 461 to `row.operator('moco.addmovementaxis', text = 'Add New MoCo Axis', icon = 'ZOOM_IN')`

It looks like these 4 changes made this awesome add-on compatible with Blender 3.0.0
I will keep you posted!

Thanks and all the best,
Max

**Comment by Defaultio** [comment]
Thanks for the fixes and thanks for using the Add-on! Will commit these changes to the main branch.

**Comment by Queerdog** [comment]
Hi, sorry, while playing with it, I solved two more issues:

1. Updating a scene has changed in newer versions from bpy.context.scene.update() to `bpy.context.view_layer.update()`
This needs to be changed in two lines (sorry my lines changed, since I added some other stuff.

2. When restarting Blender the Scene does not update, because the Handler is not loaded.
-> Solution: Make `def animationUpdate(scene):` persistent

- Add `@persistent` just before
- Import persisten handler in the top `from bpy.app.handlers import persistent`

Now it works quite well and no issues so far.
While I am starting to get a hang of Phython, I really should also learn how to use Github though ;)

Thanks for doing the heavy lifting!

Best,
Max
